invenio_indexer/cli.py

```diff
@@ -52,3 +53,19 @@ def run(delayed, concurrency):
     else:
         click.secho("Indexing records...", fg='green')
         RecordIndexer().process_bulk_queue()
+
+
+@index.command()
```

Comment:
Probably to be chainable, it's needed a PR in search module, because the [@index.group()](https://github.com/inveniosoftware/invenio-search/blob/master/invenio_search/cli.py#L47) is there